packages/alert/src/index.js

```diff
@@ -0,0 +1,83 @@
+//@flow
+import type { StatelessFunctionalComponent, Node } from "react";
+import type { AlertType } from "./AlertTypes";
+import React from "react";
+import styled from "styled-components";
+import AlertTypes from "./AlertTypes";
+
+type AlertComponent = StatelessFunctionalComponent<{
+  text: string,
+  type?: AlertType,
+  dismissable?: boolean,
+  onDismiss?: Function
+}>;
+const Alert: AlertComponent = ({
+  text,
+  type = AlertTypes.NEWS,
+  dismissable = true,
+  onDismiss = null
+}) => {
+  const buttonAttributes = onDismiss ? { onClick: onDismiss } : {};
+  const dismissButton = dismissable ? (
+    <ThemedDismissButton {...buttonAttributes} />
+  ) : null;
+  const props = { text, dismissButton, type };
+  return <ThemedBaseAlert {...props} />;
+};
+
+type BaseAlertComponent = StatelessFunctionalComponent<{
+  text: string,
+  dismissButton: Node,
+  className: string
+}>;
+const BaseAlert: BaseAlertComponent = ({ text, dismissButton, className }) => {
```

Comment:
I think we should have only one component per file.
So... Move this component to its own file.